data/definitions/animations.yml

```diff
@@ -1,4 +1,7 @@
 # Dialogue expressions
+
+expression_unknown_expression: 9807
```

Comment:
Is this the correct unknown id?

Reply:
no that was just so i can know its wrong and need to come back and fix it i didn't want to put up a wrong expression

Reply:
Have you got the actual id? I'll see if I can find an equivilant

Reply:
no i don't sorry